src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -335,7 +339,7 @@ def __read_kv_from_config_store(azconfig_client,
                                                                              accept_datetime=datetime,
                                                                              fields=query_fields)
     except HttpResponseError as exception:
-        raise CLIError('Failed to read key-value(s) that match the specified key and label. ' + str(exception))
+        raise HTTPError('Failed to read key-value(s) that match the specified key and label. ' + str(exception))
```

Comment:
Should this be `AzureResponseError`?

Reply:
You are right, I think `HTTPResponseError`s should raise `AzureResponseError`